tests/client/core/TestAddonManager.js

```diff
@@ -36,6 +36,16 @@ describe('addonManager', () => {
     };
   });
 
+  describe('hasAddonManager', () => {
+    it('does if mozAddonManager is in navigator', () => {
+      assert.ok(addonManager.hasAddonManager({ navigator: { mozAddonManager: {} } }));
+    });
+
+    it('does not if mozAddonManager is not in navigator', () => {
```

Comment:
Plus, the suggestion of @kumar303 has the advantage of not being doubled negated. Double negation tends to be harder to understand.